### PR TITLE
Add column to display resource related to a schedule

### PR DESCRIPTION
**Pull request by nixocio**
Add column to display what resource is related to a schedule

This change was added based on the feedback from @trahman73 .

image

See: https://github.com/ansible/awx/issues/5012

### CODE REVIEW

**Review by akus062381 — COMMENTED** [review]
nothing is broken but I wanted to bring something to your attention-

when a user creates a schedule inside a project or a job template, and they click the link to those resources from the list of schedules, it navigates to the detail pages for those resources. 
but- when a user clicks the resource link to a management job from the list of schedules, it doesn't navigate to the details page for that management job. it navigates to the list of management jobs.
So technically there isn't anything wrong, but perhaps for continuity, you could have them work the same way?

File: awx/ui/src/components/Schedule/ScheduleList/ScheduleList.js [hunk 1]
```diff
@@ -166,7 +166,8 @@ function ScheduleList({
         headerRow={
           <HeaderRow qsConfig={QS_CONFIG}>
             <HeaderCell sortKey="name">{t`Name`}</HeaderCell>
-            <HeaderCell sortKey="unified_job_template__polymorphic_ctype__model">{t`Type`}</HeaderCell>
+            <HeaderCell>{t`Related resource`}</HeaderCell>
```
Comment (AlexSCorey) [inline awx/ui/src/components/Schedule/ScheduleList/ScheduleList.js]: Lets add a sort key here
```suggestion
            <HeaderCell sortKey="unified_job_template">{t`Related resource`}</HeaderCell>
```

**Review by AlexSCorey — CHANGES_REQUESTED** [review]
Great job. Lets improve the UX by adding sort functionality for the new column.

**Review by akus062381 — APPROVED** [review]

**Comment by nixocio** [comment]
> nothing is broken but I wanted to bring something to your attention-
> 
> when a user creates a schedule inside a project or a job template, and they click the link to those resources from the list of schedules, it navigates to the detail pages for those resources. but- when a user clicks the resource link to a management job from the list of schedules, it doesn't navigate to the details page for that management job. it navigates to the list of management jobs. So technically there isn't anything wrong, but perhaps for continuity, you could have them work the same way?

@akus062381, good observation. However management jobs do not have a details page. That is why.

File: awx/ui/src/components/Schedule/ScheduleList/ScheduleList.js (hunk 1 above)
Comment (nixocio) [reply awx/ui/src/components/Schedule/ScheduleList/ScheduleList.js]: Done.

**Comment by nixocio** [comment]
@AlexSCorey , PR updated ready to be reviewed again.

**Review by AlexSCorey — APPROVED** [review]
Looks good to me.

**Pull request merged**